script/home-assistant@.service

```diff
@@ -0,0 +1,20 @@
+# This is a simple service file for systems with systemd to tun HA as user.
```

Comment:
Please don't add any service files to the script directory. If this is for vagrant, add it to the vagrant folder.